application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -78,8 +78,10 @@
         &lt;dt&gt;&lt;label&gt;$services.localization.render("${class.name}_${prop.name}")&lt;/label&gt;&lt;/dt&gt;
         &lt;dd&gt;
           &lt;select id="stepOrderSelect"&gt;
-            #foreach($i in [0..$mathtool.sub($nbSteps, 1)])
-              &lt;option value="$i" #if ($vobj.getProperty('order').value == $i) selected #end &gt;$mathtool.add($i, 1)&lt;/option&gt;
+            #if ($nbSteps &gt; 0)
+              #foreach($i in [0..$mathtool.sub($nbSteps, 1)])
+                &lt;option value="$i" #if ($vobj.getProperty('order').value == $i) selected #end &gt;$mathtool.add($i, 1)&lt;/option&gt;
```

Comment:
Valid XHTML requires that the `selected` attribute be in full form, i.e. `selected="selected"`